pipeline-marketplace-images/github-sign-off/create-issue

```diff
@@ -2,29 +2,56 @@
 
 set -euo pipefail
 
-if [ -f /kratix/metadata/auto-approve ]; then
-  echo "Auto-approve enabled. Skipping issue creation..."
-  exit 0
-fi
+function create_issue() {
+  if [ -f /kratix/metadata/skip-approval ]; then
+    echo "Skipping approval process as per configuration..."
+    return 2
+  fi
 
-currentRequest=$(yq -r '.status.approvalIssue' /kratix/input/object.yaml)
-if [ "$currentRequest" != "null" ]; then
-  echo "Approval issue already exists (or auto-approved). Skipping issue creation..."
-  exit 0
-fi
+  if [ -f /kratix/metadata/auto-approve ]; then
+    echo "Auto-approve enabled. Skipping issue creation..."
+    return 2
+  fi
+
+  currentRequest=$(yq -r '.status.approvalIssue' /kratix/input/object.yaml)
+  if [ "$currentRequest" != "null" ]; then
+    echo "Approval issue already exists (or auto-approved). Skipping issue creation..."
```

Comment:
Is this an "or auto-approved"? I thought that was handled in a different block?

Reply:
I think its behaving same as before. Here if the request *is* set in the status, we can recheck its status. if its not set then we need to create the issue again